### DISCUSSION TITLE
23 gennaio 2023 B - esercizio 3

### DISCUSSION THREAD

**Discussion by IonutCicio** [sapienzastudentsnetwork/basi-di-dati-1#28] [Esami]
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/40034429/6e904393-47d0-4074-b8e4-6c2d264fba87)

**Comment by niccolo-fato** [comment]
![IMAGE 2024-01-15 10:02:23](https://github.com/sapienzastudentsnetwork/bdi2324/assets/73995334/c3f44d40-10f2-4317-b000-2dc464fe545c)

**Comment by atyion** [comment]
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/101365532/28953848-38c7-4530-8132-ac60b82fbe6b)

**Comment by Warcophyr** [comment]
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/133346488/bf1c3636-f11e-492a-bb0f-e824e7523339)
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/133346488/0016fd65-df11-4af9-8f7b-bcd032b4126c)
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/133346488/827ce85f-5d75-4c95-a395-b875e1019f2f)
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/133346488/bdbc691d-60b1-418a-be9a-4e8d4f8b04fa)
